lib/hammer_cli_foreman/trend.rb

```diff
@@ -0,0 +1,45 @@
+module HammerCLIForeman
+
+  class Trend < HammerCLIForeman::Command
+
+    resource :trends
+
+    class ListCommand < HammerCLIForeman::ListCommand
+      output do
+        field :id, _("Id")
+        field :label, _("Name")
```

Comment:
I think this should be `:name` field as it shows the name of a trend. The same for `InfoCommand`.

Reply:
Do you think it should be :name although the name field is optional?